clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h

```diff
@@ -205,6 +205,19 @@ class CIRBaseBuilderTy : public mlir::OpBuilder {
     return createSub(lhs, rhs, false, true);
   }
 
+  struct CheckedBinOpResults {
+    mlir::Value result;
+    mlir::Value overflow;
+  };
+
+  CheckedBinOpResults createCheckedBinOp(mlir::Location loc,
```

Comment:
You should do the same for `CheckedBinOpResults` and `createCheckedBinOp`

Reply:
Updated.